src/main/java/slimeknights/tconstruct/smeltery/TinkerSmeltery.java

```diff
@@ -92,12 +104,51 @@
   });
   public static final RegistryObject<TileEntityType<TankTileEntity>> tank = TILE_ENTITIES.register("tank", TankTileEntity::new, (set) -> set.addAll(searedTank.values()));
   public static final RegistryObject<TileEntityType<FaucetTileEntity>> faucet = TILE_ENTITIES.register("faucet", FaucetTileEntity::new, searedFaucet);
+  public static final RegistryObject<TileEntityType<CastingBasinTileEntity>> basin = TILE_ENTITIES.register("basin", CastingBasinTileEntity::new, castingBasin);
+  public static final RegistryObject<TileEntityType<CastingTableTileEntity>> table = TILE_ENTITIES.register("table", CastingTableTileEntity::new, castingTable);
 
   /*
    * Items
    */
   public static final ItemObject<Item> searedBrick = ITEMS.register("seared_brick", SMELTERY_PROPS);
+  public static final ItemObject<Item> blankCast = ITEMS.register("blank_cast", SMELTERY_PROPS);
+  public static final ItemObject<Item> ingotCast = ITEMS.register("ingot_cast", SMELTERY_PROPS);
+  public static final ItemObject<Item> nuggetCast = ITEMS.register("nugget_cast", SMELTERY_PROPS);
+  public static final ItemObject<Item> gemCast = ITEMS.register("gem_cast", SMELTERY_PROPS);
+  public static final ItemObject<Item> pickaxeHeadCast = ITEMS.register("pickaxe_head_cast", SMELTERY_PROPS);
```

Comment:
For now, would be better to add only casts for items we have, so pickaxe head, small binding, and tool rod. We will be removing some of the old tools and parts later and I would rather not need migrations on ForgeCraft.

You can leave the textures/models if you want, most we will need later. Just want to avoid adding items we do not need for now.

Reply:
As an update on this, maybe just comment out casts other than those in alex's PR. So broadsword (except the sword cross guard, that one is likely removed), pickaxe and shovel. I can also link you the WIP parts doc if you dont have it, will show which parts still exist

Reply:
Did so.